tests/test_infer_ops/triton/test_context_attn_unpad.py

```diff
@@ -0,0 +1,156 @@
+import pytest
+import torch
+import torch.nn.functional as F
+from packaging import version
+
+from colossalai.kernel.triton import context_attention_unpadded
+from colossalai.utils import get_current_device
+
+try:
```

Comment:
this is always true?

Reply:
I think it might be removed by pre-commit

Reply:
Try add `# noqa` to the import statement.

Reply:
Import statement revised.